tests/test_urban_mental_health.py

```diff
@@ -0,0 +1,717 @@
+                                               path_to_shp, projection_wkt,
+                                               "ESRI Shapefile", fields,
+                                               attribute_list)
+
+
+def make_raster_from_array(base_raster_path, array):
+    """Create a raster on designated path with array values.
+
+    Args:
+        base_raster_path (str): the raster path for making the new raster.
+
+    Returns:
+        None.
+    """
+    # UTM Zone 10N
+    srs = osr.SpatialReference()
+    srs.ImportFromEPSG(26910)
+    projection_wkt = srs.ExportToWkt()
+
+    pixel_size = (100, -100)
+    no_data = FLOAT32_NODATA
+
+    pygeoprocessing.numpy_array_to_raster(
+        array.astype(numpy.float32), no_data, pixel_size,
+        (ORIGIN_X, ORIGIN_Y), projection_wkt, base_raster_path)
+
+
+def make_synthetic_data_and_params(workspace_dir):
+    """Make all data needed to run UMH model"""
+
+    # make synthetic input data
+    baseline_prevalence_path = os.path.join(
+        workspace_dir, "baseline_prevalence.shp")
+    fields = {"id": ogr.OFTReal, "risk_rate": ogr.OFTReal}
+    attribute_list = [{"id": 0, "risk_rate": 10}]
+    make_simple_vector(baseline_prevalence_path, fields, attribute_list,
+                       shapely_geometry_list=[
+                           Polygon([(ORIGIN_X, ORIGIN_Y-250),
+                                    (ORIGIN_X+250, ORIGIN_Y-250),
+                                    (ORIGIN_X+250, ORIGIN_Y),
+                                    (ORIGIN_X, ORIGIN_Y),
+                                    (ORIGIN_X, ORIGIN_Y-250)])])
+
+    ndvi_base_array = numpy.array(
+        [[.1, .2, .35], [.5, .6, .7],
+         [.8, .9, .10], [.11, .12, FLOAT32_NODATA]])
+    ndvi_base_path = os.path.join(workspace_dir, "ndvi_base.tif")
+    make_raster_from_array(ndvi_base_path, ndvi_base_array)
+
+    ndvi_alt_array = numpy.array(
+        [[.12, .22, .1], [.2, .3, .8], [.9, .14, .14], [.16, .17, .3]])
+    ndvi_alt_path = os.path.join(workspace_dir, "ndvi_alt.tif")
+    make_raster_from_array(ndvi_alt_path, ndvi_alt_array)
+
+    pop_array = ndvi_alt_array*100
+    pop_path = os.path.join(workspace_dir, "population.tif")
+    make_raster_from_array(pop_path, pop_array)
+
+    aoi_path = os.path.join(workspace_dir, "aoi.shp")
+    make_simple_vector(aoi_path)
+
+    args = {
+        'aoi_path': aoi_path,
+        'baseline_prevalence_vector': baseline_prevalence_path,
+        'effect_size': 0.94,
+        'health_cost_rate': None,
+        'lulc_alt': '',
+        'lulc_attr_csv': '',
+        'lulc_base': '',
+        'ndvi_alt': ndvi_alt_path,
+        'ndvi_base': ndvi_base_path,
+        'population_raster': pop_path,
+        'results_suffix': 'test1',
+        'scenario': 'ndvi',
+        'search_radius': 100, # 1 pixel
+        'tc_raster': '',
+        'tc_target': '',
+        'workspace_dir': workspace_dir,
+    }
+
+    return args
+
+
+gdal.UseExceptions()
```

Comment:
This feels like an odd spot for this... If this is necessary can we move this just after imports?